File: 240329/길이 단위 변환하기/change-length-unit.js
function ftToCm(value){
    return value * 30.48;
}

function miToCm(value){
    return value * 160934;
}

let ft = 9.2;
let mi = 1.3;

console.log(`${ft}ft = ${ftToCm(ft).toFixed(1)}cm`);
console.log(`${mi}mi = ${miToCm(mi).toFixed(1)}cm`);

File: 240329/길이 단위 변환하기/change-length-unit.java
public class Main {
    static final double FT = 30.48;
    static final int MI = 160934;

    public static void main(String[] args) {
        double ftValue = 9.2;
        double miValue = 1.3;

        System.out.printf("%.1fft = %.1fcm\n", ftValue, ftToCm(ftValue));
        System.out.printf("%.1fmi = %.1fcm", miValue, miToCm(miValue));
    }
    
    static double ftToCm(double value){
        return FT * value;
    }

    static double miToCm(double value){
        return MI * value;
    }
}